HTML draft

Lujoffily opened the pull request:
I did the HTML with the IDs, buttons and etc. base on the wireframe. If possible give me feedback on if there is anything I should take away or add or improve.

Ryan9698 reviewed (approved):
Thanks! I made some small changes but it was based more on preference so I removed most of the hyphens and changed class names to a different format, (Generator-btn -> genBtn). That's mostly for me though so I don't confuse myself. Thank you! - Ryan

Ryan9698 commented:
Thank you for the push! Enjoy your trip! - Ryan

Lujoffily commented:
> Thanks! I made some small changes but it was based more on preference so I removed most of the hyphens and changed class names to a different format, (Generator-btn -> genBtn). That's mostly for me though so I don't confuse myself. Thank you! - Ryan

>* Okay perfect! No worries at all. Let me know if there is anything I can help with on JS.
-Luna
